src/stock-items/add-stock-item/packaging-units/packaging-units.component.tsx

```diff
@@ -179,6 +183,23 @@ const PackagingUnitRow: React.FC<{
 }> = ({ isEditing, row, key }) => {
   const { t } = useTranslation();
 
```

Comment:
lets stick o3 implementation style of modals and dialogs

Reply:
> lets stick o3 implementation style of modals and dialogs

This too